pkg/networkservice/common/mechanisms/kernel/server.go

```diff
@@ -39,6 +39,7 @@ func NewServer() networkservice.NetworkServiceServer {
 func (m *kernelMechanismServer) Request(ctx context.Context, request *networkservice.NetworkServiceRequest) (*networkservice.Connection, error) {
 	if mechanism := kernel.ToMechanism(request.GetConnection().GetMechanism()); mechanism != nil {
 		mechanism.SetNetNSURL((&url.URL{Scheme: "file", Path: netNSFilename}).String())
+		mechanism.SetInterfaceName(request.GetConnection().GetId())
```

Comment:
Why is this setting the interface to the connection id where above it was setting it to the ${ns}-${connectionID} ?

Reply:
Sure, fixed.
${ns}-${connectionID} is used for consistency.